hack/gcb_testing/simulate_build_trigger.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Skaffold Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IMAGE=$1
+BUCKET=$2
+
+gcloud container builds submit . \
+    --config=deploy/cloudbuild.yaml \
+    --substitutions="_SKAFFOLD_IMAGE=$IMAGE,_SKAFFOLD_BUCKET=$BUCKET,COMMIT_SHA=testing_$(date +%s)"
```

Comment:
I'd rather have this be driven off the makefile rather than cloudbuild substitutions.

Reply:
I am not sure what you mean - I create a target in the Makefile, but the substitutions would stay anyway - unless we want to test the pipelines only in prod.